hyperdash_cli/cli.py

```diff
@@ -128,6 +134,85 @@ def demo(args=None):
 
     exp.end()
 
+
+def github(args=None):
+    port = _find_available_port()
+    if not port:
+        print("Github sign in requires an open port, please open port 3000.")
+
+    # Signal when the HTTP server has started
+    server_started_queue = Queue()
+    # Signal when we have the access token
+    access_token_queue = Queue()
+
+    # Server that we will run in the background to accept a post-OAuth redirect from
+    # the Hyperdash server which will contain the user's access token
+    def start_server():
+        class OAuthRedirectHandler(BaseHTTPServer.BaseHTTPRequestHandler):
+            def do_GET(self):
+                parsed_path = urlparse(self.path)
+                query = parse_qs(parsed_path.query)
+                access_token = query["access_token"][0]
+                access_token_queue.put(access_token)
+                # Redirect user's browser
+                self.send_response(301)
+                self.send_header('Location',"{}/{}".format(get_base_http_url(), "/oauth/github/success"))
```

Comment:
Use consistent quotes.

Reply:
A linter is the systematic answer here. 

@richardartoul @spence Do you guys have a preference for a python linter? My constraint is that I want VSCode's python plugin to support it, leaving us with these three options:
https://www.pylint.org/
https://pypi.python.org/pypi/pep8
https://flake8.readthedocs.io/en/latest/